src/api/index.js

```diff
@@ -343,6 +358,82 @@ export function getInstallProperties() {
   return get(uri);
 }
 
+
+export async function patchServiceAccount(
+  serviceAccountName,
```

Comment:
Can we use named parameters instead of positional ones? This is consistent with most of our existing APIs and generally tends to produce more readable and flexible code. So this function would accept an object as a parameter instead:

`... patchServiceAccount({ serviceAccountName, namespace, secretName }) { ...`


Same for `getServiceAccount` above